src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsList.jsx

```diff
@@ -22,7 +22,13 @@ class ViewDependentsList extends Component {
     return (
       <>
```

Comment:
I wasn't able to verify since my local environment isn't set up to look at dependents, but I think returning a fragment here instead of a div leads to undivided, flat elements for both lists. So instead of having :
```
<div class="container">
  <div>
    <h2>On award</h2>
// list content...
  </div>
  <div>
    <h2>Not on award</h2>
// list content
  </div>
</div>
```
You might instead have:
```
<div class="container">
  <h2>On award</h2>
// list content...
  <h2>Not on award</h2>
// list content
</div>
```
This is probably more of a opinion on semantics though 🤷‍♂